.github/workflows/build-on-ubuntu.yml

```diff
@@ -17,7 +17,6 @@ jobs:
         with:
           java-version: 8
           distribution: zulu
-          cache: gradle
```

Comment:
Why do we turn the cache off?

Reply:
@alexander-yevsyukov 
That's because it causes the build to fail sometimes, as it uses the cached `.class` files from the previous workflow runs. I don't want to take any chance with that, especially when publishing the library.